packages/preview-middleware-client/src/cpe/init.ts

```diff
@@ -9,9 +9,16 @@ import { SelectionService } from './selection';
 import { ChangeService } from './changes/service';
 import { loadDefaultLibraries } from './documentation';
 import Log from 'sap/base/Log';
+import { enableTelemetry } from '@sap-ux-private/control-property-editor-common';
 
 export default function init(rta: RuntimeAuthoring): Promise<void> {
     Log.info('Initializing Control Property Editor');
+    
+    // enable telemetry if requested
+    const flexSettings = rta.getFlexSettings();
+    if (flexSettings.telemetry === true) {
+        enableTelemetry();
```

Comment:
Similar call needs to be also done in the `control-property-editor` React app.

Reply:
I haven't looked at that part of the code at all, could you please take of it?